src/gui/tray/Window.qml

```diff
@@ -442,6 +521,13 @@ Window {
                                         color: appEntry.hovered ? Style.lightHover : "transparent"
                                     }
                                 }
+
+                                Accessible.role: Accessible.MenuItem
+                                Accessible.name: appName
+                                Accessible.description: qsTr("Opens the corresponding Nextcloud app in a new browser window")

```

Comment:
Name could be "Open `appName` in browser" and then the description is not needed.